ReadMe.md

```diff
@@ -90,9 +90,12 @@ English | Japanese | Kana | Notes
 --------|----------|------|------
 Development | 開発 | かいはつ |
 Developer | 開発者 | かいはつしゃ |
+Engineer | PG (プログラマ) | ぷろぐらまー | 
```

Comment:
I simply chose Engineer there because some people prefer the word Engineer over Programmer.
It seemed that more people prefer Engineer.

By the way, it seems that Japanese engineers are usually called programmers.

Anyways, I can separate them like so:

| Programmer | PG | プログラマー |
| エンジニア | PG/エンジニア | エンジニア |

Let me know what you think of it.